op-deployer/pkg/deployer/integration_test/apply_test.go

```diff
@@ -613,6 +619,126 @@ func TestApplyGenesisStrategy(t *testing.T) {
 	}
 }
 
+func TestProofParamOverrides(t *testing.T) {
+	op_e2e.InitParallel(t)
+
+	lgr := testlog.Logger(t, slog.LevelDebug)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	depKey := new(deployerKey)
+	l1ChainID := big.NewInt(77799777)
+	dk, err := devkeys.NewMnemonicDevKeys(devkeys.TestMnemonic)
+	require.NoError(t, err)
```

Comment:
Seems all the tests in this file have a lot of boilerplate that gets duplicated, would be nice to clean that up a bit (can be a separate PR or this one)

Reply:
Will do as a separate PR; would like to reorder some stuff in the test file too that will make a big diff.